cartopy/meta.yaml

```diff
@@ -10,8 +10,8 @@ source:
     - cartopy.win.patch  # [win]
 
 build:
-  number: 0  # [not linux]
-  number: 1  # [linux]
+  number: 1  # [not linux]
+  number: 2  # [linux]
```

Comment:
If we bump both to release 2 then the channel would not have release 1 for OSX and Windows.  It is a minor inconsistency (and I do that in the IOOS channel all the time), but that does not seems to be the way @pelson do things here.